Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM odisseus/ideprobe-pants:202.6948.69
+FROM odisseus/ideprobe-pants-candidate:202.7319.50
```

Comment:
this is the place I hoped we can replace version with latest, so that we don't have to change it (limit places where we need to change on intellij bump.

Reply:
Ok